### DISCUSSION TITLE
react flow multigraph

### DISCUSSION THREAD

**Discussion by pharreX19** [xyflow/xyflow#1858] [Q&A]
In the normal flow the two edges from NODE A towards NODE B follows the same path, however is there anyway to separate their paths or each edge takes a distinctive path? Im using dagre for layouting, and i can see that dagre can detect 2 edges yet they still follows single path (on top of each other)? does anyone know how to fix this issue ?

**Comment by vaibhav-systango** [comment]
@moklick any help here ?

**Reply by moklick** [reply]
The solution is to create a custom node with multiple handles (each for every connection) as shown in this example https://reactflow.dev/docs/examples/nodes/custom-node/
